<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUERY</class>
 <widget class="QDialog" name="QUERY">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>547</width>
    <height>337</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>547</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>学生信息查询</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>请输入查询学生的学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请输入查询学生的姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>501</width>
       <height>192</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>192</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
